### PR TITLE
Moving read logic to dsv2-common, no logic changes

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: spark-bigquery-dsv2/spark-bigquery-dsv2-common/src/main/java/com/google/cloud/spark/bigquery/v2/context/BigQueryIndirectWriterCommitMessageContext.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (VinayLondhe14) [inline spark-bigquery-dsv2/spark-bigquery-dsv2-common/src/main/java/com/google/cloud/spark/bigquery/v2/context/BigQueryIndirectWriterCommitMessageContext.java:1]: Will changes for the data source writer be created in a new PR?

Comment (davidrabinowitz) [reply spark-bigquery-dsv2/spark-bigquery-dsv2-common/src/main/java/com/google/cloud/spark/bigquery/v2/context/BigQueryIndirectWriterCommitMessageContext.java:1]: Yes - smaller PRs (as much as 23 files are involved...) are easier to review. The writer will be in a subsequent PR.

**Review by VinayLondhe14 — APPROVED** [review]

**Pull request merged**